### PR TITLE
My first pull request - Alejandro Rivas

### PR DESCRIPTION
**Pull request by aerr96**
## Class 2 Week 1 exercise:

1.  I made the fork directly from the GitHub page of the undefined-academy repository to my user repositories.
2.  Then, I cloned the repository to a folder called "semana1" in my root path. Just using the command  `` git clone <my_forked_repo_url> ``
3.  I moved to the directory `` github-profiles `` using the command `` cd ``, I created the README file with the command `` touch `` with the name of my discord username.
4.  Using the command `` code `` I edited the README file with my previously made Markdown code in VisualStudio Code.
5. With the command, `` git add `` I added the file to the repository, and then with the command `` git commit `` I committed.
6. Finally, I'm requesting a pull request to finish and review this first exercise, thanks so much.

📝**Note**: Indiferentemente de que el curso es dictado en Español, prefiero redactar mis informes y ejercios en ingles, ya que, busco entrar directamente a un mercado laboral en ingles. Gracias.

### CODE REVIEW

**Comment by aerr96** [comment]
### Class 3 Week 1 exercise: 

1. Highlight last class missing step: after commit requires a push with `` git push `` 
2. Folder created with `` mkdir `` command.
3. Previous `` .md `` file copied to a new folder as a new file and named README.
4. Index HTML file created according to the class 3 and committed all the changes.

**Comment by aerr96** [comment]
### Class 4 Week 1 final commit:

1. Following Jeremy Thomas tutorial, a ccs style file was implemented, adding formats, color, and an image to reach a better-looking presentation.

File: github-profiles/aerr96-4844.md
Comment (dfm555) [inline github-profiles/aerr96-4844.md:1]: Mueve este archivo dentro de carpeta aerr96-4844/ o elimínalo pues no deben ir archivos dentro de github-profiles/

File: github-profiles/aerr96-4844/README.md
Comment (dfm555) [inline github-profiles/aerr96-4844/README.md:1]: Agrega los alias

File: github-profiles/aerr96-4844/style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    padding-top: 1em;
+}
+
+h1,
+h2,
+strong {
+    color: #333;
+}
+
+/*code,
```
Comment (dfm555) [inline github-profiles/aerr96-4844/style.css:63]: Elimina el código comentado

**Review by dfm555 — CHANGES_REQUESTED** [review]

File: github-profiles/aerr96-4844/style.css (hunk 1 above)
Comment (aerr96) [reply github-profiles/aerr96-4844/style.css]: Listo acabo de limpiar y ajustar mi PR con los cambios sugeridos, gracias. Espero la re-revision.

**Pull request closed without merging**